extension/app/ts/background/metadataUtils.ts

```diff
@@ -119,7 +119,7 @@ async function getTokenMetadata(simulator: Simulator, address: bigint) : Promise
 	if (nftTokenData) return {
 		name: nftTokenData.name,
 		symbol: nftTokenData.symbol,
-		logoURI: nftTokenData.logoURI ? `${ LOGO_URI_PREFIX }/${ nftTokenData.logoURI }` : undefined,
+		logoURI: nftTokenData.logoUri ? `${ LOGO_URI_PREFIX }/${ nftTokenData.logoUri }` : undefined,
```

Comment:
I thought we renamed these to logoRelativePath?

Reply:
Ya i just had this all setup already and didn't want to change so much in a single pass https://github.com/DarkFlorist/address-metadata/issues/10